brainrender/scene.py

```diff
@@ -452,10 +452,15 @@ def add_root(self, render=True, **kwargs):
 		:param **kwargs:
 
 		"""
+		print(brainrender.ROOT_COLOR)
 		if not render:
-			self.root = self.atlas._get_structure_mesh('root', c=brainrender.ROOT_COLOR, alpha=0, **kwargs)
+			print("there")
```

Comment:
I like these changes, though we need to remove the print statements ;)
also there's better ways to debug code :P

Reply:
:D sorry for this and the previous change, I forgot to revert before pushing :p